### PR TITLE
Add crossmint to dev docs /dapps and /enterprise

### LINKED ISSUE #11636: Adding Crossmint to the list of Enterprise Developer Resources
### Dev tool name

Crossmint

### Dev tool description

Crossmint is the leading enterprise-grade web3 development platform, allowing you to build NFT applications without requiring any blockchain experience or holding cryptocurrency, and making the blockchain invisible to end users. The platform offers a one-stop-shop for most of your NFT infrastructure needs across all major chains, including creating, distributing, selling, storing digital assets, and offering post-mint utility. 

### When did the dev tool go live?

February, 2022

### Where do you think this page should be shown?

https://ethereum.org/en/enterprise/#enterprise-developer-resources
Under this page, under "Products and services".

https://ethereum.org/en/enterprise/#enterprise-live-on-mainnet
Under this page, under "Payments"

### Is the product open source?

No

### GitHub URL

https://github.com/Crossmint

### Documentation URL

https://docs.crossmint.com/

### Additional context

_No response_

### Would you like to work on this issue?

- [x] Yes
- [ ] No

```[tasklist]
### Tasks
- [ ] https://github.com/ethereum/ethereum-org-website/pull/11684
- [ ] https://github.com/ethereum/ethereum-org-website/pull/11683
- [ ] https://github.com/ethereum/ethereum-org-website/pull/11685
```

### PR DESCRIPTION
**Pull request by rohit-710**
Adding Crossmint on Ethereum Docs

## Description

Updating Tools for Creating DAPPS to include Crossmint and Updating Ethereum Mainnet for enterprise doc to show Crossmint

## Related Issue
- Closes #11636

### CODE REVIEW

**Comment by rohit-710** [comment]
https://github.com/ethereum/ethereum-org-website/issues/11636

Issue link

**Review by nnnnnnsssss — APPROVED** [review]

**Comment by tasdienes** [comment]
I think that Crossmint should only be listed under *one* category, not under multiple categories.

File: src/content/enterprise/index.md
Comment (wackerow) [inline src/content/enterprise/index.md:1]: I'm not sure I see other products that we're listing multiple times on this page. The number of items to choose from throughout this page is already quite unwieldy...

Requesting we choose one category that best fits what Crossmint offers.

**Review by wackerow — CHANGES_REQUESTED** [review]
@rohit-710 Left a request. Also, we remove `utm_source` trackers from all links if you could please make that adjustment. Thanks!

File: src/content/enterprise/index.md
Comment (rohit-710) [reply src/content/enterprise/index.md:1]: Hey @wackerow 
I just updated this

**Review by wackerow — APPROVED** [review]

**Pull request merged**